app/services/process_interaction_service.rb

```diff
@@ -47,7 +47,7 @@ def call(envelope, target_account)
         reflect_unblock!(account, target_account)
       end
     end
-  rescue HTTP::Error, OStatus2::BadSalmonError, Mastodon::NotPermittedError
+  rescue HTTP::Error, OStatus2::BadSalmonError, Mastodon::NotPermittedError, Mastodon::WorkerError
```

Comment:
It's for deal with rspec error. WorkerError will be raised from inside of perform_async only when test.